outer.h

```diff
@@ -24,6 +24,8 @@ struct outer_helper {
 	pid_t pid;
 	int in;
 	int out;
+
+	int cleanfd;
```

Comment:
Same comment as earlier, cleanfd tells the reader nothing. This is a cgroup fd that only incidentally ends up being cleaned up.

Reply:
Switched to socket IPC.